templates/form/form-element-label.html.twig

```diff
@@ -10,11 +10,11 @@
   set classes = [
     'ecl-form-label',
     title_display == 'after' ? 'option',
-    title_display == 'invisible' ? 'visually-hidden ecl-form-label--hidden',
+    title_display == 'invisible' ? 'visually-hidden',
     required ? 'js-form-required',
     required ? 'form-required',
   ]
 %}
 {% if title is not empty or required -%}
-  <label{{ attributes.addClass(classes) }}>{{ title }}</label>
+  <label{{ attributes.addClass(classes) }}{{ title_display == 'invisible' ? ' hidden ' }}>{{ title }}</label>
```

Comment:
Not needed space after `hidden ': 
![image](https://user-images.githubusercontent.com/153362/65128131-273e7700-d9f9-11e9-849e-8ef4629ab723.png)

Reply:
done.